packages/chrome-extension/src/singleton.ts

```diff
@@ -45,7 +45,7 @@ export async function buildClerk({
   // Set up cookie params based on environment
   const getClientCookieParams: GetClientCookieParams = isProd
     ? {
-        urls: key.frontendApi,
+        urls: `https://${key.frontendApi}`,
```

Comment:
Can we merge the logic we have here and in `ensureFormattedUrl` somehow? 
For example, use here the `ensureFormattedUrl` function.

Reply:
@anagstef `ensureFormattedUrl` is going to handle a lot more use-cases soon. This PR is largely to solve a singular current issue.